speaker/main.go

```diff
@@ -72,6 +72,7 @@ func main() {
 		myNode     = flag.String("node-name", os.Getenv("METALLB_NODE_NAME"), "name of this Kubernetes node (spec.nodeName)")
 		port       = flag.Int("port", 7472, "HTTP listening port")
 		logLevel   = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", strings.Join(logging.Levels, ", ")))
+		bgpType    = flag.String("bgp-implementation-type", "native", "Specifies the BGP implementation type.")
```

Comment:
Can we defer this to be part of a patch series that makes the new option do something?

Reply:
Sure. I was looking for early feedback on this but what I can do is split this PR in two and make the second one (i.e. the configuration option) a draft. I'll do this later in the week.

Reply:
From our dev community call yesterday, i think we decided to change this to an env var instead of a config option, at least while it's still in development.  Then I'm fine putting it in early since it will be more hidden while it's still WIP.

Reply:
Ok, I will still split the PR so I will remove this commit.